src/components/spreadsheet/utils/config-tables.js

```diff
@@ -1682,7 +1682,7 @@ export const TABLES_DEFINITIONS = {
                     return {
                         defaultValue:
                             params.data?.activePowerControl
-                                ?.activePowerControlOn | 0,
+                                ?.activePowerControlOn,
```

Comment:
You should set the value to an empty string instead of undefined to avoid receiving a warning in the console
```suggestion
                                ?.activePowerControlOn || '',
```

Reply:
If we include it, the default value won't be displayed when editing if it's set to "false."